helpers/error_handler/error_message.go

```diff
@@ -0,0 +1,32 @@
+package helpers
```

Comment:
Because I want to use alias package importing, something like this
```go
import error_handler "github.com/gutakk/go-google-scraper/helpers/error_handler"
error_handler.ValidationErrorToText(...)

// instead of
import "github.com/gutakk/go-google-scraper/helpers"
helpers.ValidationErrorToText(...)
```
or
```go
import session "github.com/gutakk/go-google-scraper/helpers/session"
session.Set(...)

// instead of
import "github.com/gutakk/go-google-scraper/helpers"
helpers.Set(...)
```

IMO, this will cause confusion if we don't use alias. What do you think?

Reply:
I agree that using aliases is better in the context of controllers. Sinc in Go, the import is for whole packages then using a directory is needed.